site/es/guide/keras/functional.ipynb

```diff
@@ -0,0 +1,1519 @@
+   "metadata": {
+    "colab_type": "text",
+    "id": "HoMFNu-pnkgF"
+   },
+   "source": [
+    "Lo que se devuelve, ` input `, contiene información sobre la forma y el tipo de dato que se espera ingresa en tu modelo:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "ddIr9LPJnibj"
+   },
+   "outputs": [],
+   "source": [
+    "inputs.shape"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "lZkLJeQonmTe"
+   },
+   "outputs": [],
+   "source": [
+    "inputs.dtype"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "kZnhhndTnrzC"
+   },
+   "source": [
+    "Puedes crear un nuevo nodo en el grafico de capas mandando a llamar al objeto ` input `."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "sMyyMTqDnpYV"
+   },
+   "outputs": [],
+   "source": [
+    "from tensorflow.keras import layers\n",
+    "\n",
+    "dense = layers.Dense(64, activation='relu')\n",
+    "x = dense(inputs)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "besm-lgFnveV"
+   },
+   "source": [
+    "La acción \"layer call\" es como dibujar una flecha desde \"inputs\" a la capa que creamos.\n",
+    "Estamos \"pasando\" las entradas a la capa `dense`, y afuera obtenemos` x`.\n",
+    "\n",
+    "Agreguemos algunas capas más a nuestro gráfico de capas:\n",
+    "\n",
+    "La acción \"llamada a la capa\" es como dibujar una flecha de \"Inputs\" a la capa que creamos.\n",
+    "\n",
+    "Estamos pasando las inputs a una capa mas densa, y respecto a la salida obtenemos una ` x `."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "DbF-MIO2ntf7"
+   },
+   "outputs": [],
+   "source": [
+    "x = layers.Dense(64, activation='relu')(x)\n",
+    "outputs = layers.Dense(10, activation='softmax')(x)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "B38UlEIlnz_8"
+   },
+   "source": [
+    "LLegados a este punto, podemos crear un ` Modelo ` especificando sus entradas y salidas en las capas de graficas."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "MrSfwvl-nx9s"
+   },
+   "outputs": [],
+   "source": [
+    "model = keras.Model(inputs=inputs, outputs=outputs)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "5EeeV1xJn3jW"
+   },
+   "source": [
+    "To recap, here is our full model definition process:\n",
+    "Recapitulando, esta es nuestra definción completa del proceso:"
+   ]
```

Comment:
"To recap, here is our full model definition process no eliminaste la linea en ingles